.github/workflows/main.yml

```
@@ -7,23 +7,16 @@ jobs:
     runs-on: ubuntu-18.04
     name: Evaluator job
     steps:
-      - name: Wait for build to succeed
-        uses: betrybe/action-wait-for-check@v2
-        id: wait-for-build
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          checkName: codeclimate
-          ref: ${{ github.event.pull_request.head.sha || github.sha }}
       - uses: actions/checkout@v2
-        if: steps.wait-for-build.outputs.conclusion == 'success'
       - name: Cypress evaluator step
         id: cypress
-        uses: betrybe/cypress-evaluator-action@v1
-        if: steps.wait-for-build.outputs.conclusion == 'success'
+        uses: betrybe/cypress-evaluator-action@v2
```

```suggestion
        uses: betrybe/cypress-evaluator-action@v4
```

Ja utiliza a última versão por favor (https://github.com/betrybe/cypress-evaluator-action/releases/tag/v4)